docs/_data/sidebars/mydoc_sidebar.yml

```diff
@@ -54,4 +54,12 @@ entries:
       url: /admin_guide.html
       output: web
 
+  - title: Releases
```

Comment:
how does this page look

Reply:
Here is a gist page : https://gist.github.com/bvaradar/4f320b64d034cf67da78f5c640261bd6